src/Main.java

```diff
@@ -1,9 +1,36 @@
+import java.util.Scanner;
 
 public class Main {
     public static void main(String[] args) {
 
-        // Hier mag je je code schrijven voor de hoofd-opdracht
+        // Declare variables //
+        Integer[] numeric = {1, 2, 3, 4, 5, 6, 7, 8, 9, 0};
+        String[] alphabetic = {"één", "twee", "drie", "vier", "vijf", "zes", "zeven", "acht", "negen", "nul"};
+        boolean play = true;
+        String invalid = "ongeldige invoer";
 
+        // Declare objects //
+        Translator translateNumToAlpha = new Translator(numeric, alphabetic);
+        Scanner inputScanner = new Scanner(System.in);
+
+        // Decision structure number translator //
+        while (play) {
+            System.out.println("Type 'x' om te stoppen / Type 'v' om te vertalen.");
+            String input = inputScanner.nextLine();
+            if (input.equals("x")) {
+                play = false;
+            } else if (input.equals("v")) {
+                System.out.println("Type een cijfer in van 0 t/m 9.");
+                int number = inputScanner.nextInt(); // Als je nu een letter invoert crashed hij. Hoe kun je dit oplossen?
+                inputScanner.nextLine(); // Waarom deze extra nextLine?? Bij niet toevoegen crashed hij....
```

Comment:
nextInt is een apart geval in Java. Zie dit artikel voor meer informatie; https://www.freecodecamp.org/news/java-scanner-nextline-call-gets-skipped-solved/